topiary-cli/Cargo.toml

```diff
@@ -28,6 +28,8 @@ path = "src/main.rs"
 # Eventually we will want to dynamically load them, like Helix does.
 clap = { workspace = true, features = ["derive"] }
 env_logger = { workspace = true }
+futures = { workspace = true }
+tree-sitter-facade = { workspace = true }
```

Comment:
Can we avoid this dependency if we let the library convert from `topiary::Language`?

Reply:
It was really only for the type, which we can infer with `_`. However, since the app is the foremost client-app for the library, we might still want to do this.